app/src/main/res/values/strings.xml

```diff
@@ -70,6 +70,9 @@
   <string name="patiententry_error_phonenumber_length_less">Phone number cannot be less than 6 digits</string>
   <string name="patiententry_error_phonenumber_length_more">Phone number cannot be more than 12 digits</string>
   <string name="patiententry_identifier">BP Passport</string>
+  <string name="patiententry_consent_header">Consent</string>
+  <string name="patiententry_consent_first_para">: By tapping below, you confirm that any patient data you enter here has been obtained under appropriate informed consent. This means that the patient knows that you are entering their personal data into the app, understands what data is being collected, and knows that they may be contacted via SMS, WhatsApp, or other methods using the phone number provided in the app.</string>
```

Comment:
I am guessing you made `: ` part of this string because of the space in the end.
You can add an explicit space using Unicode for space.
`:\u0020`

Reply:
No this part of the design. The colon is not bold.

Reply:
I checked with Pragati, the colon is bold.
Can you make this change?

Reply:
If it's not much problem, please make this change and approve it yourself :)

Reply:
Sure.